kola/tests/misc/verity.go

```diff
@@ -96,8 +96,8 @@ func VerityCorruption(c cluster.TestCluster) {
 	// get usr device, probably vda3
 	usrdev := util.GetUsrDeviceNode(c, m)
 
-	// write zero bytes to first 10 MB
-	c.MustSSH(m, fmt.Sprintf(`sudo dd if=/dev/zero of=%s bs=1M count=10 status=none`, usrdev))
+	// write zero bytes to first 100 MB
+	c.MustSSH(m, fmt.Sprintf(`sudo dd if=/dev/zero of=%s bs=1M count=100 status=none`, usrdev))
```

Comment:
the flush can be done directly by dd `oflag=dsync`

Reply:
On a decent NVME:

```
$: time dd if=/dev/zero of=t bs=1M count=100 status=none oflag=dsync

real    0m0.114s
user    0m0.000s
sys     0m0.057s

$: time dd if=/dev/zero of=t bs=1M count=100 status=none

real    0m0.057s
user    0m0.000s
sys     0m0.053s
```

Reply:
yes

Reply:
Merging the commands would also work when combined with `&&` to keep the error checking for the first one. When settings flags for `dd` to remove the `sync` we would just have to do enough tests to make sure that the test doesn't become flaky, or we could also keep the extra `sync` just to be sure.